tasties_app/views.py

```diff
@@ -9,12 +9,16 @@
 
 
 def base(request):
-    return render(request, 'tasties_app/base.html',)
+    return render(request, 'tasties_app/base.html')
 
 
 @login_required(login_url='login')
 def recipes(request):
-    recipes_list = Recipe.objects.all()
+    if request.method == 'POST':
+        search_value = request.POST.get('search')
+        recipes_list = Recipe.objects.filter(title__icontains=search_value)
```

Comment:
I think  Recipe.objects.filter(title=search_value) it enough but if it work like that it ok also.

Reply:
mmm maybe you right but also i was think if there word contain in some one of recipes and i find this in google also is work good and can check uppercase  and lowercase .